examples/ddi_seal/main.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import os
+import sys
+
+import torch
+import torch.nn as nn
+import warnings
+from config import get_cfg_defaults
+from model import SAGE
+from kale.utils.seed import set_seed
+from ogb.linkproppred import PygLinkPropPredDataset, Evaluator
+from dataloader import SEALDataset, SEALDynamicDataset
+from torch_geometric.data import DataLoader
+from trainer import Trainer
+from scipy.sparse import SparseEfficiencyWarning
+warnings.simplefilter('ignore', SparseEfficiencyWarning)
+
+
+def arg_parse():
+    """Parsing arguments"""
+    parser = argparse.ArgumentParser(description='PyTorch CIFAR10 Training')
+    parser.add_argument('--cfg', required=True, help='path to config file', type=str)
+    parser.add_argument('--output', default='results', help='folder to save output', type=str)
+    parser.add_argument('--data_appendix', type=str, default='')
```

Comment:
Google what does data appendix mean. I do not feel this understandable.

Reply:
the appendix means the stored enclosing subgraph's file name suffix, which is used for describe constructed subgraph's attributes on this.